pkg/scraper/script.go

```diff
@@ -207,3 +203,66 @@ func findPythonExecutable() (string, error) {
 
 	return "python3", nil
 }
+
+func handleStderrLine(line string, defaultLogLevel *log.Level) {
+	level, l := log.DetectLogLevel(line)
+
+	const scraperPrefix = "[Scrape] "
+	// if no log level, just output to info
+	if level == nil {
+		if defaultLogLevel != nil {
+			level = defaultLogLevel
+		} else {
+			level = &log.InfoLevel
+		}
+	}
+
+	switch *level {
+	case log.TraceLevel:
+		logger.Trace(scraperPrefix, l)
+	case log.DebugLevel:
+		logger.Debug(scraperPrefix, l)
+	case log.InfoLevel:
+		logger.Info(scraperPrefix, l)
+	case log.WarningLevel:
+		logger.Warn(scraperPrefix, l)
+	case log.ErrorLevel:
+		logger.Error(scraperPrefix, l)
+		// case log.ProgressLevel:
+		// 	progress, err := strconv.ParseFloat(l, 64)
+		// 	if err != nil {
+		// 		logger.Errorf("Error parsing progress value '%s': %s", l, err.Error())
+		// 	} else {
+		// 		// only pass progress through if channel present
+		// 		if s.progress != nil {
+		// 			// don't block on this
+		// 			select {
+		// 			case s.progress <- progress:
+		// 			default:
+		// 			}
+		// 		}
+		// 	}
```

Comment:
Progress shouldnt be needed when scraping and would be more complex to implement. Just delete the comments imo